thématique/gc-thématique-en.html

```diff
@@ -93,17 +94,35 @@ <h2>List of GC promotional feature projects</h2>
 		</ul>
 		{% endif %}
 	</li>
+	{% endif %}
 {% endfor %}
 </ul>
 
 <h2>GC promotional thematic meta file</h2>
 
 <pre><code>---
 feature: thématique
-codename: Canada Day 2021
-expiry: June 2022
+lang: en
+title: Title of the thematic usage
+description: Short description of the added style
+componentName: th-canadaday
+expiry: June 30, 2022
+mainPage: canada-day.html
 cssClass:
-  - bg-canadaday
+- List of CSS class
+jsFunctions:
+- List of javascript function
+a11yStatement: >
+  Accessibility statement, describe what was tested, include any notes/instructions for implementers to follow in order to maintain the accessibility conformance, during and after the thematic expiration.
+  Tested by (Name of the person Nick Frenette, nick.frenette at canada.ca. 2021-06-01.
+peNote:
+- Progressive enhancement notes and instructions
+pages:
+  examples:
+    - title: Title of the page
+      language: en
+      path: index.html
```

Comment:
Same for the example here, even though it has 0 impact: `canada-day.html`.

Reply:
This is just an example of what a configuration file might look like. We would need to add more documentation later on and do some change at our current build script

Reply:
I know, that's why I said it has 0 impact. However, to be consistent with the actual meta file I strongly suggested to change it here too.

Reply:
I removed the Canada day reference